tests/conftest.py

```diff
@@ -453,6 +472,45 @@ def duthost(duthosts, request):
     return duthost
 
 
+@pytest.fixture(name="dpuhosts", scope="session")
+def fixture_dpuhosts(enhance_inventory, ansible_adhoc, tbinfo, request):
+    """
+    @summary: fixture to get DPU hosts defined in testbed.
+    @param ansible_adhoc: Fixture provided by the pytest-ansible package.
+        Source of the various device objects. It is
+        mandatory argument for the class constructors.
+    @param tbinfo: fixture provides information about testbed.
+    """
+    try:
+        host = DutHosts(ansible_adhoc, tbinfo, get_specified_dpus(request),
```

Comment:
Need add a request in the param list.
https://github.com/sonic-net/sonic-mgmt/blob/1918f01dae8263075a3450fa161543252dbbc1ec/tests/common/devices/duthosts.py#L50

Reply:
Done